### PR TITLE
[polaris.shopify.com] Add support for markdown content in component examples' descriptions

### PR DESCRIPTION
**Pull request by alex-page**
Some descriptions have markdown that wasn't getting parsed. This fixes this.

### CODE REVIEW

File: polaris.shopify.com/src/components/Examples/Examples.tsx [hunk 1]
```diff
@@ -73,7 +74,11 @@ const Examples = (props: Props) => {
         selected={options[currentIndex].value}
         onChange={handleSelection}
       />
-      {description ? <p>{description}</p> : null}
+      {description ? (
+        <p>
+          <Markdown text={description} />
+        </p>
```
Comment (martenbjork) [inline polaris.shopify.com/src/components/Examples/Examples.tsx]: You don't need the wrapping `<p>`.

image

e61ad5868c49b6e5535510259984685d

**Review by martenbjork — APPROVED** [review]

File: .changeset/odd-wolves-attack.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'polaris.shopify.com': minor
+---
+
+Example descriptions support for markdown content
```
Comment (martenbjork) [inline .changeset/odd-wolves-attack.md]: > Added support for markdown content in component examples' descriptions

**Review by martenbjork — APPROVED** [review]

File: .changeset/odd-wolves-attack.md (hunk 2 above)
Comment (alex-page) [inline .changeset/odd-wolves-attack.md]:
```suggestion
Added support for markdown content in component examples' descriptions
```

File: polaris.shopify.com/src/components/Examples/Examples.tsx (hunk 1 above)
Comment (alex-page) [inline polaris.shopify.com/src/components/Examples/Examples.tsx]:
```suggestion
        <Markdown text={description} />
```

**Pull request merged**